Dockerfile

```diff
@@ -0,0 +1,11 @@
+# This is a development Dockerfile. For versioned Dockerfiles see:
+# https://github.com/praekeltfoundation/docker-seed
+FROM praekeltfoundation/django-bootstrap:py2.7-jessie
+
+ COPY . /app
+RUN pip install -e git+git://github.com/onaio/onapie.git#egg=onapie &&\
+    pip install -e .
+
+ ENV DJANGO_SETTINGS_MODULE "malaria24.settings"
```

Comment:
We also typically use a specific settings file that has docker specific settings. See [molo-gem example](https://github.com/praekeltfoundation/molo-gem/blob/develop/Dockerfile#L4).

Reply:
And the related settings file looks like [this](https://github.com/praekeltfoundation/molo-gem/blob/develop/gem/settings/docker.py)

Reply:
I will add the settings file